### PR TITLE
added universal button component

### PR DESCRIPTION
**Pull request by krayzieeddi**
I have added a universal button component to be used across the whole website and I only implemented the new button component to the home page just to ease it in rather than go all the way just in case of unforeseen issues

### CODE REVIEW

**Review by lgtanimoto — COMMENTED** [review]
Looks good.  What other pages have to be modified using this component?

File: src/components/button.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+// this universal button component can be used as a general buttom template to be use across the whole app
+// You do not need to use every prop here for the button to work
+const Button = ({ id, onClick, buttonText, type }) => {
```
Comment (brianyamasaki) [inline src/components/button.js]: I have a trick for you that'll make this feel more like a real button. You can remove "buttonText" and replace it with "children". Then when somebody types &lt;Button onclick={() =>{}}&gt;&lt;icon id="arrow" /&gt;Submit&lt;/Button&gt;, both the icon component and the text Submit will come in as the children. Replace {buttonText} below with {children} and you're good to go. 

The best part of this is you can add styling to this button and every button will look identical. Or have a style component that can accept OK buttons vs. Cancel Buttons.

**Review by brianyamasaki — COMMENTED** [review]
I've left an idea to make your Button component almost identical to the HTML button component. Please take a look.

File: src/components/button.js (hunk 1 above)
Comment (krayzieeddi) [reply src/components/button.js]: @lgtanimoto I only used the button component for the home.js page

Comment (krayzieeddi) [reply src/components/button.js]: have now changed the buttonText to children. Thank you Brian for the info.

**Comment by krayzieeddi** [comment]
I only used the button component on the home.js page

**Review by lgtanimoto — COMMENTED** [review]
Thanks for clarification

**Pull request merged**